bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -43,6 +43,7 @@
 #include "FakeClock.hpp"
 #include <ccron/ticks_generator.hpp>
 #include "EpochManager.hpp"
+#include "DbCheckpointManager.hpp"
```

Comment:
not reqruied, please see the comment below

Reply:
removed